english/warning-ignore/ignore.md

```diff
@@ -3,3 +3,8 @@ to a rustling sound. Deciding to investigate you go to the other room only
 to find it filled with rats. There is a moment of silence when the rats stop
 moving and stare at you. Then suddenly, as quickly as a flash of lightening
 the rats begin to swarm you.
+
+Do you:
+- [grab the nearest item to hand and throw it at the rats?](rat-fight/flaming_rats.md)
+- [turn and run out the room?](rat-fight/ice-berg.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.